PWGDQ/Core/HistogramsLibrary.cxx

```diff
@@ -531,6 +531,52 @@ void o2::aod::dqhistograms::DefineHistograms(HistogramManager* hm, const char* h
         hm->AddHistogram(histClass, "Mass_QuadDCAsigXYZ", "", false, 50, 0.0, 5.0, VarManager::kMass, 50, 0.0, 20.0, VarManager::kQuadDCAsigXYZ);
         hm->AddHistogram(histClass, "Mass_Pt_QuadDCAsigXYZ", "", false, 500, 0.0, 5.0, VarManager::kMass, 100, 0.0, 10.0, VarManager::kPt, 50, 0.0, 20.0, VarManager::kQuadDCAsigXYZ);
       }
+      if (subGroupStr.Contains("largescale")) {
+        // binning for mee at large scales:
+        // every 10 MeV from 0 to 1.1 GeV/c2
+        // every 50 MeV from 1.1 to 2.7 GeV/c2
+        // every 10 MeV from 2.7 to 3.2 GeV/c2
+        // every 50 MeV from 3.2 to 10 GeV/c2
+        double mee_bins[329];
```

Comment:
do you need such high mee? mee upto 10 GeV/c2 seems too much. keep it up to 5 GeV/c2.

Reply:
See comment above.